platform/linux-generic/include/odp/api/plat/packet_types.h

```diff
@@ -67,101 +67,89 @@ typedef enum {
 
 #endif
 
-/** @internal Packet field accessor */
+/** @cond _ODP_HIDE_FROM_DOXYGEN_ */
+
```

Comment:
This a section label for doxygen. We'll never add it to doxygen config or define it otherwise. I think it just need to high light that this portion of a file intentionally left hiden from doxygen documentation. "Internal" does not tell that this is only a doxygen label, it would seem like a  configuration option or a preprocessor macro.

Reply:
OK